core/tests/protractor/explorationFeedback.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for feedback on explorations
+ * Tests the following sequence:
+ * User 1 creates and publishes an exploration.
+ * User 2 plays the exploration and leaves feedback on it
+ * User 1 reads the feedback and responds to it.
+ * Can't test what happens after that because User 2 can only
```

Comment:
I think this could do with a little more elaboration. Suggest replacing with:

```
FYI for developers: In production, after this sequence of events, the only notification User 2 will receive is via email, and we can't easily test this in an e2e test.
```